lodash.src.js

```diff
@@ -3592,6 +3594,15 @@
         }
         return fn.apply(thisBinding, args);
       }
+
+      if (isAryExact) {
+        var plist = [];
+        for (var i = 0; i < ary; i++) { plist.push('p' + i); }
+        var params = plist.join(',');
+        var fn = wrapper;
+        eval('wrapper = function(' + params + ') { return fn.apply(this, arguments); }');
```

Comment:
Using `eval` won’t work in CSP enabled environments, like Chrome extensions. We won’t be able to consistently set the function arity because of this.